doc/lua_api.md

```diff
@@ -259,7 +259,18 @@ time, if necessary. (See [`Settings`])
 
 Media files (textures, sounds, whatever) that will be transferred to the
 client and will be available for use by the mod and translation files for
-the clients (see [Translations]).
+the clients (see [Translations]). Accepted characters for names are:
+
+    a-zA-Z0-9_.-
+
+Accepted formats are:
+
+    images: .png, .jpg, .bmp, .tga
```

Comment:
maybe add a note that tga is pending deprecation?

I don't want to ruin your simple PR with a big discussion but this has to be considered when documenting this for the first time.